src/Fixer/PhpUnit/PhpUnitNamespacedFixer.php

```diff
@@ -32,6 +32,11 @@ final class PhpUnitNamespacedFixer extends AbstractFixer implements Configuratio
      */
     private $originalClassRegEx;
 
+    /**
+     * @var array|string[]
+     */
+    private $originalClassTranslations;
```

Comment:
please add reasoning for this array in description
like why we need to have it in first place. for now, it's hidden knowledge why some classes are put here and some are not

Reply:
thanks for pointing to that, I removed the indirection and made it a more direct old => new class-map now and explained that it assists the regex which provides the foremost class detection and helps out for those cases where it matches but *"do[es] not have a same-compound namespace class and need a dedicated translation table."*. Also referencing *`configure()`* method which now has the array configuration below the regex nicely paired.